internal/serialization/object_data_test.go

```diff
@@ -472,20 +455,19 @@ func TestPositionalObjectDataOutput_PWriteUint16(t *testing.T) {
 	var expected = uint16(32)
 	o.PWriteUInt16(15, expected)
 	i := NewObjectDataInput(o.buffer, 0, nil, false)
-	res := i.ReadUInt16WithPosition(15)
-	assert.NoError(t, i.err)
+	res := i.ReadUInt16AtPosition(15)
 	assert.Equal(t, res, expected)
 }
 
 func TestPositionalObjectDataOutput_PWriteBool(t *testing.T) {
 	o := NewPositionalObjectDataOutput(100, nil, false)
 	o.PWriteBool(15, true)
 	i := NewObjectDataInput(o.buffer, 0, nil, false)
-	res := i.ReadBoolWithPosition(15)
-	assert.NoError(t, i.err)
+	res := i.ReadBoolAtPosition(15)
 	assert.Equal(t, res, true)
 }
 
+/*
```

Comment:
nit: `t.SkipNow()` should be used to skip tests. If this test is no longer valid, it's better to remove it completely